services/grpc/grpcService.js

```diff
@@ -77,8 +77,15 @@ class GrpcService extends EventEmitter {
   /**
    * Log successful connection.
    */
-  async onAfterConnect() {
-    await this.subscribe()
+  onAfterConnect() {
+    // Wait for 2 seconds before subscribing to grpc streams.
+    // This gives lnd a chance to start up it's services after unlocking a wallet.
+    // We add this delay as in recent versions fo lnd (post 0.6.0), the channel stream is not functional immediately.
+    setTimeout(() => {
```

Comment:
can we use `await delay(2000)` here?

Reply:
The reason I didn't use await delay is because I want this to run in the background. If we await, it will hold up the progress of the state transition.

Reply:
Fair enough. In that case, just for the sake of not using `setTimeout` and callbacks, how do you feel about this code then 
```
function onAfterConnect() {
     // non blocking delay
    (async function () {
        await delay(2000)
        if (this.is('connected')) {
            this.subscribe()
          }
    })()
    //do stuff, no block 
}
```

Reply:
I'v adjusted this